src/service/calculator/calculator_repository.py

```diff
@@ -71,6 +108,30 @@ def get_budget_options(self, year: int, need_prior: bool, need_next: bool) -> li
                     self.get_metric_option(
                         "Budget", "Ebitda", "prior_budget_ebitda", prior_year
                     ),
+                    self.get_metric_option(
+                        "Budget",
+                        "Cost of good",
+                        "prior_budget_cost_of_goods",
+                        prior_year,
+                    ),
+                    self.get_metric_option(
+                        "Budget",
+                        "Sales & marketing",
+                        "prior_budget_sales_marketing_cost",
+                        prior_year,
+                    ),
+                    self.get_metric_option(
+                        "Budget",
+                        "General & administration",
+                        "prior_budget_general_admin_cost",
+                        prior_year,
+                    ),
+                    self.get_metric_option(
+                        "Budget",
+                        "Research & development",
+                        "prior_budget_research_development_cost",
+                        prior_year,
+                    ),
```

Comment:
We don't need these options.
Please rename the function to get_forward_budget_options

Reply:
these were deleted and the funcion was renamed.